README.md

```diff
@@ -1,90 +1,130 @@
+12 | Feb 25 | Hudson Jameson | [agenda](https://github.com/flashbots/pm/issues/16) | [slides](https://docs.google.com/presentation/d/1I0cTl74p3MF_ZSyfjoH4A3YC6nJkK3_Ot302jiNPs5I/edit#slide=id.gbfd4392ce8_0_5) | [video](https://drive.google.com/file/d/1RLWCdvd47PHLN_az762r3G00BsnBumnb/view?usp=sharing) |
 
-#### Resources to write a MEV searcher:
+## Resources
+* [Flashbots: Frontrunning the MEV Crisis (medium)](https://medium.com/flashbots/frontrunning-the-mev-crisis-40629a613752)
+* [Flashbots: Frontrunning the MEV crisis (EthR post)](https://ethresear.ch/t/flashbots-frontrunning-the-mev-crisis/8251)
+* [Quantifying MEV: Introducing MEV-Explore v0](https://medium.com/flashbots/quantifying-mev-introducing-mev-explore-v0-5ccbee0f6d02)
+* [explore.flashbots.net](https://explore.flashbots.net)
+* [Flashbots Discord](https://discord.gg/7hvTycdNcK)
 
-- Example arbitrage searcher: https://github.com/flashbots/simple-arbitrage
-- Flashbots ethers.js provider: https://github.com/flashbots/ethers-provider-flashbots-bundle
-- Searcher FAQ: https://github.com/flashbots/pm/blob/main/guides/flashbots-alpha.md
-- Dedicated Flashbots Discord channel [#🤖searchers](https://discord.gg/KNFBvZzJyT).
+## Navigate Flashbots' Github
+[/pm](https://github.com/flashbots/pm)
+<br> The repo you're currently in, contains an introduction to our organization, our job postings and the list of all the projects we're working on, updated frequently by the project's owners.
 
-See you on chain ! :zap::robot: 
+[/mev-geth](https://github.com/flashbots/mev-geth)
+<br> A fork of the Geth client dubbed MEV-Geth modified to accept Flashbots transaction bundles and to compare Flashbots blocks to regular blocks.
 
+[/mev-geth-demo](https://github.com/flashbots/mev-geth-demo)
+<br> Launches an MEV GETH node, and shows how a miner may profit from it by accepting MEV bundles
 
-## Contribute to MEV-Research
+[/mev-relay-js](https://github.com/flashbots/mev-relay-js)
+<br> The code for the entrypoint searchers send their Flashbots bundles to and miners connect to in order to receive bundles. Includes an example of a a reverse proxy that a miner can run.
 
-We invite you to review our [MEV-Research](https://github.com/flashbots/mev-research) repo to learn about our MEV Fellowship program. Start contributing through opening or answering a Github issue, and/or writing a Flashbots Research Proposal (FRP) to co-author our research papers and/or apply for a research grant, and join our discussion on our [MEV-Research discord](https://discord.gg/9yuuZendEy) community.
+[/ethers-provider-flashbots-bundle](https://github.com/flashbots/ethers-provider-flashbots-bundle)
+<br> Flashbots provider for ethers.js
 
-## Join the MEV Roast!
+[/web3-flashbots](https://github.com/flashbots/web3-flashbots)
```

Comment:
I'd describe this and the previous one with the same wording, since they both provide alternative approaches to the same thing.